src/layer/vector/SVG.js

```diff
@@ -17,7 +17,7 @@ export var create = Browser.vml ? vmlCreate : svgCreate;
  * Allows vector layers to be displayed with [SVG](https://developer.mozilla.org/docs/Web/SVG).
  * Inherits `Renderer`.
  *
- * Due to [technical limitations](http://caniuse.com/#search=svg), SVG is not
+ * Due to [technical limitations](https://caniuse.com/#search=svg), SVG is not
```

Comment:
We can use a direct link to the feature here.
```suggestion
 * Due to [technical limitations](https://caniuse.com/svg), SVG is not
```